### ISSUE TITLE
Cannot read property '_c' of undefined

### ISSUE THREAD

**Issue by cxxwl96** [leezng/vue-json-pretty#141]
when i use at first it, and it whill be broken with "Cannot read property '_c' of undefined"

**Comment by cxxwl96** [comment]
```
Uncaught (in promise) TypeError: Cannot read property '_c' of undefined
    at Proxy.<anonymous> (vue-json-pretty.js:formatted:1)
    at renderComponentRoot (runtime-core.esm-bundler.js:1167)
    at componentEffect (runtime-core.esm-bundler.js:5220)
    at reactiveEffect (reactivity.esm-bundler.js:42)
    at effect (reactivity.esm-bundler.js:17)
    at setupRenderEffect (runtime-core.esm-bundler.js:5173)
    at mountComponent (runtime-core.esm-bundler.js:5132)
    at processComponent (runtime-core.esm-bundler.js:5090)
    at patch (runtime-core.esm-bundler.js:4684)
    at mountChildren (runtime-core.esm-bundler.js:4880)
```

**Comment by cxxwl96** [comment]
> when i use at first it, and it whill be broken with "Cannot read property '_c' of undefined"

I used vue3 and vue-json-pretty used version 1.8.1, which was fixed when I updated to 2.0.4
